### PR TITLE
[stable-2.9] Escalate priv before setting terminal

### PR DESCRIPTION
**Pull request by NilashishC**
(cherry picked from commit 0cd2ad588032ac1d15ca7084c5c03c91e607a80a)

##### SUMMARY
Backport of https://github.com/ansible/ansible/pull/63571/

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
network_cli.py

##### ADDITIONAL INFORMATION
- Fixes https://github.com/ansible-collections/cisco.nxos/issues/170

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

File: changelogs/fragments/escalate_priv_before_terminal.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Perform privilege escalation before setting terminal.
```
Comment (relrod) [inline changelogs/fragments/escalate_priv_before_terminal.yaml]:
```suggestion
  - network_cli connection plugin - Perform privilege escalation before setting terminal.
```

**Pull request merged**